_topic_map.yml

```diff
@@ -1568,6 +1568,8 @@ Topics:
       File: cnv-installing-qemu-guest-agent
     - Name: Viewing the IP address of NICs on a virtual machine
       File: cnv-viewing-ip-of-vm-nic
+    - Name: Enabling KubeMacPool for virtual machines 
```

Comment:
I don't think that the name of the component matters that much. Maybe it should be "MAC address pooling for virtual machines"?

Reply:
Absolutely. I've been rearranging this a bit throughout the afternoon to make the headings  more useful.